lib/models/schema.js

```diff
@@ -17,6 +17,24 @@ const MixinSpecificationExtensions = require('../mixins/specification-extensions
  * @returns {Schema}
  */
 class Schema extends Base {
+  constructor (json) {
+    const isBoolSchema = typeof json === 'boolean';
+
+    let jsonValue = json;
+    if (isBoolSchema) {
+      if (json === true) {
+        // {} is a sugar syntax for `true` Boolean Schema
+        jsonValue = {};
+      } else {
+        // { not: {} } is a sugar syntax for `false` Boolean Schema
+        jsonValue = { not: {} };
```

Comment:
I would not expect the parser to handle this tbh... It would be confusing when my boolean all of the sudden disappeared from `_json`.

Reply:
I wrote about it in the comment in discussion

> Also in current changes I passed for `true` schema the `{}` model body and for `false` the `{ not: {} }` and by this we have two ways for handling Boolean Schemas.

In some situation it's good, in another not 😄

Reply:
In general, we should never restructure/change the original JSON by changing the core representation of it. It seems wrong from a parser's perspective.

Reply:
Done, I also removed `booleanValue`, because we can use for it the normal `json` function.